include/uapi/linux/tee.h

```diff
@@ -43,15 +43,15 @@
 #define TEE_IOC_BASE	0
 
 /* Flags relating to shared memory */
-#define TEE_IOCTL_SHM_MAPPED	0x1	/* memory mapped in normal world */
-#define TEE_IOCTL_SHM_DMA_BUF	0x2	/* dma-buf handle on shared memory */
+#define TEE_IOCTL_SHM_MAPPED	BIT(0)/* memory mapped in normal world */
```

Comment:
The problem with this is that the `BIT()` macro isn't available for userspace and this file is shared with userspace (it's under `include/uapi`).

Reply:
Indeed. I ran into this issue and was wondering what you'd suggest doing. There are a few (very few) other headers under `include/uapi` that use the `BIT` macro.

I also noticed that the version of `tee.h` in the OP-TEE Client repository is fairly different from the one in the kernel, so I defined the macro there if it's not already been defined elsewhere. This of course doesn't help any other users of this header in user-space.

I'd vote for making these all shifts; `checkpatch` will complain, but at least the header is consumable by everyone.